sdks/ios/ZelloChannelDemo/ZelloChannelDemo/Views/RiderTalkViewController.m

```diff
@@ -10,6 +10,7 @@
 
 #import "RiderTalkViewController.h"
 #import "common.h"
+#import "Zello.h"
 
 @interface RiderTalkViewController () <ZCCSessionDelegate>
 @property (nonatomic, strong) IBOutlet UIButton *buttonCancel;
```

Comment:
Not something you added, maybe update all of these to be weak so they are in line with the one you added